plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -311,6 +347,10 @@ public void onOutputLine(String line) {
       } else if (line.equals("Traceback (most recent call last):")) {
         // An error occurred
         setServerState(IServer.STATE_STOPPED);
+      } else if (serverPort == 0 && line.contains("Starting module") && line.contains("running at: ")) {
```

Comment:
This doesn't work for the multi-service case; it's ok to embed "_default_" here since it's a well-known service id.  Here's what I see for a multi-service run:

```
INFO     2016-11-04 03:16:39,769 devappserver2.py:769] Skipping SDK update check.
INFO     2016-11-04 03:16:39,875 api_server.py:205] Starting API server at: http://localhost:61831
INFO     2016-11-04 03:16:39,882 dispatcher.py:197] Starting module "default" running at: http://localhost:8080
INFO     2016-11-04 03:16:39,886 dispatcher.py:197] Starting module "second" running at: http://localhost:8081
INFO     2016-11-04 03:16:39,888 admin_server.py:116] Starting admin server at: http://localhost:8000
Nov 04, 2016 3:16:41 AM com.google.appengine.tools.development.AbstractContainerService configure
WARNING: Null value for containerConfigProperties.get(devappserver.portMappingProvider)
Nov 04, 2016 3:16:42 AM com.google.apphosting.utils.jetty.JettyLogger info
Nov 04, 2016 3:16:41 AM com.google.appengine.tools.development.AbstractContainerService configure
INFO: Logging to JettyLogger(null) via com.google.apphosting.utils.jetty.JettyLogger
Nov 04, 2016 3:16:42 AM com.google.apphosting.utils.jetty.JettyLogger info
INFO: jetty-6.1.x
WARNING: Null value for containerConfigProperties.get(devappserver.portMappingProvider)
Nov 04, 2016 3:16:42 AM com.google.apphosting.utils.jetty.JettyLogger info
Nov 04, 2016 3:16:42 AM com.google.apphosting.utils.jetty.JettyLogger info
INFO: Started SelectChannelConnector@localhost:61837
Nov 04, 2016 3:16:42 AM com.google.appengine.tools.development.AbstractModule startup
INFO: Logging to JettyLogger(null) via com.google.apphosting.utils.jetty.JettyLogger
INFO: Module instance default is running at http://localhost:61837/
Nov 04, 2016 3:16:42 AM com.google.apphosting.utils.jetty.JettyLogger info
INFO: jetty-6.1.x
Nov 04, 2016 3:16:42 AM com.google.appengine.tools.development.AbstractModule startup
INFO: The admin console is running at http://localhost:61837/_ah/admin
Nov 04, 2016 3:16:42 AM com.google.apphosting.utils.jetty.JettyLogger info
INFO: Started SelectChannelConnector@localhost:61838
Nov 04, 2016 3:16:42 AM com.google.appengine.tools.development.devappserver2.DevAppServer2Impl doStart
INFO: Dev App Server is now running
Nov 04, 2016 3:16:42 AM com.google.appengine.tools.development.AbstractModule startup
INFO: Module instance second is running at http://localhost:61838/
Nov 04, 2016 3:16:42 AM com.google.appengine.tools.development.AbstractModule startup
INFO: The admin console is running at http://localhost:61838/_ah/admin
Nov 04, 2016 3:16:42 AM com.google.appengine.tools.development.devappserver2.DevAppServer2Impl doStart
INFO: Dev App Server is now running
```